Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -218,69 +202,55 @@ param (
             }
         }
     }
-    
+
+    Validate-Credential $credential
+    $userName = Get-DownLevelLogonName -fqdn $fqdn -userName $($credential.UserName)
+    $password = $($credential.Password)  
+    $psCredentialObject = New-Object pscredential -ArgumentList $userName, (ConvertTo-SecureString -String $password -AsPlainText -Force)
+   
     $machineShare = Get-MachineShare -fqdn $fqdn -targetPath $targetPath    
     $destinationNetworkPath = Get-DestinationNetworkPath -targetPath $targetPath -machineShare $machineShare
 
     Write-Verbose "machine share= $machineShare"
     Write-Verbose "destination network path= $destinationNetworkPath"
-    
-    Validate-Credential $credential
-    $userName = Get-DownLevelLogonName -fqdn $fqdn -userName $($credential.UserName)
-    $password = $($credential.Password) 
 
-    if([bool]([uri]$targetPath).IsUnc)
-    {
-        Create-DestinationDirectory -path $destinationNetworkPath
-    }
-    
-    $netExeCommand = Get-NetExeCommand
+    Create-DestinationDirectory -path $destinationNetworkPath
```

Comment:
This method was only tested for UNC target paths. Is removing the UNC target path check necessary?

Reply:
destinationNetworkPath is always a unc path